### PR TITLE
Feature: delegate actions

### PR DESCRIPTION
**Pull request by luixo**
Example: https://pr-1156---explorer-frontend-vcqilefdcq-uc.a.run.app/transactions/EAuEaQAZsgySoQ3gWEKzqLHdjd7Myzic1kZ9GpFUu5L5?network=testnet

### CODE REVIEW

File: frontend/src/translations/ru/common.json [hunk 1]
```diff
@@ -347,6 +349,9 @@
         },
         "DeleteKey": {
           "key_deleted": "Ключ удалён: ${public_key}..."
+        },
+        "delegate": {
```
Comment (shelegdmitriy) [inline frontend/src/translations/ru/common.json:353]: Let's use PascalCase here as we used before: 
```suggestion
        "Delegate": {
```

File: backend/src/utils/actions.ts [hunk 2]
```diff
@@ -1,68 +1,86 @@
...
+  | FunctionCallAction
+  | TransferAction
+  | StakeAction
+  | AddKeyAction
+  | DeleteKeyAction
+  | DeleteAccountAction;
+
+type DelegateAction = {
+  kind: "delegate";
+  args: {
```
Comment (khorolets) [inline backend/src/utils/actions.ts:78]: From what I can gather the list of arguments should be bigger, including:
- sender_id
- receiver_id
- actions +
- nonce
- max_block_height
- public_key

Sorry if you just skipped it during the draft and you was going to implement it later.

Comment (luixo) [reply backend/src/utils/actions.ts:78]: As we don't know what we are going to render on frontend - I didn't pass any of the props to the mapped action.

Comment (luixo) [reply backend/src/utils/actions.ts:72]: @khorolets Do you feel like we need to add anything more than we have at the moment?

**Pull request merged**